src/libYARP_dev/src/devices/ServerGrabber/ServerGrabber.cpp

```diff
@@ -888,10 +888,10 @@ void ServerGrabber::run()
                     flex_i2.setQuantum(i2.getQuantum());
                     flex_i2.resize(fgImage2->width(),fgImage2->height());
                     //ASK ma flex_i.width e height saranno 0 giusto?
-                    i.setExternal(flex_i.getRawImage(),flex_i.width(),flex_i.height());
                     fgImage->getImage(i);
-                    i2.setExternal(flex_i2.getRawImage(),flex_i2.width(),flex_i2.height());
                     fgImage2->getImage(i2);
+		    memcpy(flex_i.getRawImage(), i.getRawImage(), i.getRawImageSize());
+		    memcpy(flex_i2.getRawImage(), i2.getRawImage(), i2.getRawImageSize());
```

Comment:
Yes, don't worry. this commit is temporary (done from the robot).
It is gonna be fixed and squashed from a pc with a decent editor